fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -58,32 +83,32 @@ public ArithmeticMaker(
      */
     public ArithmeticMaker(MetricDictionary metricDictionary, ArithmeticPostAggregationFunction function) {
         // TODO: Deprecate me, mappers should always be specified at creation time, not implicitly
```

Comment:
The idea behind this TODO is that we shouldn't apply a `SketchRoundUpMapper` to everything that uses the `ArithmeticMaker`, and instead require those who _want_ that to add the mapper they want explicitly.

For example, someone might want to add 2 estimated sketches and have the result be a floating-point, rather than an integer. The Maker should not be opinionated.